app/controllers/miq_ae_class_controller.rb

```diff
@@ -1955,11 +1958,14 @@ def create_action_url(node)
 
   def get_rec_name(rec)
     column = rec.display_name.blank? ? :name : :display_name
-    if rec.kind_of?(MiqAeNamespace) && rec.domain? && (!rec.editable? || !rec.enabled)
-      add_read_only_suffix(rec, rec.send(column))
-    else
-      rec.send(column)
+    if rec.kind_of?(MiqAeNamespace) && rec.domain?
+      editable_domain = Tenant.current_tenant.editable_domains.include?(rec)
+      enabled_domain  = Tenant.current_tenant.enabled_domains.include?(rec)
```

Comment:
@h-kataria 
The Tenant.current_tenant.enabled_domains will get used by the Automate engine, we are still in the process of adding that in. I think we should use the record.enabled